spk/salt-master/src/service-setup.sh

```diff
@@ -0,0 +1,32 @@
+PYTHON_DIR="/var/packages/python38/target/bin"
+PATH="${SYNOPKG_PKGDEST}/bin:${SYNOPKG_PKGDEST}/env/bin:${PYTHON_DIR}:${PATH}"
+VIRTUALENV="${PYTHON_DIR}/python3 -m venv"
+PYTHON="${PYTHON_DIR}/python3"
+LANGUAGE="env LANG=en_US.UTF-8"
+SALT_MASTER="${SYNOPKG_PKGDEST}/env/bin/salt-master"
+PID_FILE="${SYNOPKG_PKGDEST}/env/var/run/salt-master.pid"
+
+SERVICE_COMMAND="${SALT_MASTER} -c ${SYNOPKG_PKGDEST}/env/etc/salt -d"
+
+service_postinst ()
+{
+    # Create a Python virtualenv
+    ${VIRTUALENV} --system-site-packages ${SYNOPKG_PKGDEST}/env
+
+    # Install the wheels
+    wheelhouse=${SYNOPKG_PKGDEST}/share/wheelhouse
+    ${SYNOPKG_PKGDEST}/env/bin/pip install --no-deps --no-index --upgrade --force-reinstall -f ${wheelhouse} ${wheelhouse}/*.whl
```

Comment:
This was simplified over to using standard functions instead.
Have a look at https://github.com/SynoCommunity/spksrc/wiki/Using-wheels-to-distribute-Python-packages